### ISSUE TITLE
[DISCUSSION]

### ISSUE THREAD

**Issue by tpa93** [balakethelock/SuperWoW#20]

**Comment by tpa93** [comment]
Got detected 2 times
First time as PUA:Win32/Puwaders.C!ml
Secont time as Trojan:Win32/Wacatac.A!ml
Normal?
Thanks :)

**Comment by balakethelock** [comment]
Yes its normal, it's a dll injection so antiviruses will treat it as such
